sympy/polys/tests/test_multivariate_resultants.py

```diff
@@ -0,0 +1,195 @@
+"""Tests for Dixon's and Macaulay's classes. """
+
+from sympy import Matrix
+from sympy.core import symbols
+from sympy.utilities.lambdify import lambdify
+from sympy.tensor.indexed import IndexedBase
+
+from sympy.polys.multivariate_resultants import (DixonResultant, MacaulayResultant)
+
+c, d = symbols("a, b")
+x, y = symbols("x, y")
+
+p = lambdify((x, y), c * x + y)
```

Comment:
These are not polynomials, they are Python functions.  `p = Poly(c*x + y, (x, y))` would be a polynomial.